src/main/java/io/github/bobocodebreskul/server/DispatcherServlet.java

```diff
@@ -143,6 +157,80 @@ private void processRequest(HttpServletRequest req, HttpServletResponse resp) {
     }
   }
 
+  /**
+   * Processes the method parameter.
+   *
```

Comment:
It would be really nice to see what exactly it does here as 'process' is too general.

Reply:
done